verifier/verifier.go

```diff
@@ -176,6 +175,11 @@ func (v *verifier) processSignature(ctx context.Context, sigBlob []byte, envelop
 		if err != nil {
 			return err
 		}
+        pluginVersion := metadata.Version
+        verificationPluginMinVersion, err := getVerificationPluginMinVersion(&outcome.EnvelopeContent.SignerInfo)
```

Comment:
1. Can we reuse the data of `getVerificationPluginMinVersion` call made at line 161 ?
2. Always check for error before using the output of an function

Reply:
Yes and the error from getVerificationPluginMinVersion is getting checked in line 161